src/Model/PhasorDynamics/SynchronousMachine/GENROUwS/README.md

```diff
@@ -65,7 +53,50 @@ Transformed parameters used during implementation and for readability.
 \end{aligned}
 ```
 
-## Equations
+## Model Variables
+
+### Internal Variables
+
+#### Differential
+
+Symbol      | Units   | Description                     | Note
+------------|---------|---------------------------------| ------
+$\delta$  | [rad] | Machine internal rotor angle |
+$\omega$  | [p.u.] | Machine relative speed | Optionally read by governor or stabilizer component
+$\psi'_d$ | [p.u.] | Direct axis subtransient flux | 
```

Comment:
Consider defining $\omega$ as the generator speed rather than relative speed for notation consistency.

```suggestion
$\omega$  | [p.u.] | Machine speed | Optionally read by governor or stabilizer component
$\psi'_d$ | [p.u.] | Direct axis subtransient flux | 
```